spec/mongoid/token_spec.rb

```diff
@@ -255,7 +255,7 @@ class UntaintedDocument
         it "should raise an operation failure" do
           duplicate_name = "Got Duped."
           document_class.create!(:name => duplicate_name)
-          expect{ document_class.create!(:name => duplicate_name) }.to raise_exception(Moped::Errors::OperationFailure)
+          expect{ document_class.create!(:name => duplicate_name) }.to raise_exception(Mongo::Error::OperationFailure)
```

Comment:
Line is too long. [118/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing to the left of {.